### DISCUSSION TITLE
Info on the recv_chunk_no_stop method

### DISCUSSION THREAD

**Discussion by WeisLeDocto** [LaboratoireMecaniqueLille/crappy#27] [Q&A] [answered]
Hi @Dad0u,

Do you remember why you implemented the [recv_chunk_no_stop](https://github.com/LaboratoireMecaniqueLille/crappy/blob/c756e4344bda674b94f72a25dabf7b6f77ad7531/crappy/links/link.py#L329) feature for the Recorder block ? 
Was data sometimes lost during finish ?
And you mention it was an experimental feature, did you have a chance to check if it was working properly and/or stable ?

Weis

**Comment by Dad0u** [comment, marked as answer]
Hi @WeisLeDocto,
I was indeed told that on very rare occasions, the last points of data before stopping the program could be lost. I never could reproduce the issue. I wrote this method just in case some data points got to the Recorder after receiving the stop command. The method is still called in the finish method of the Recorder block, but i did not perform enough testing to know if it is actually necessary or not.

**Reply by WeisLeDocto** [reply]
Ok pretty strange, I will try to test it a bit further whenever I have some time.
Thanks for your answer.
